test/compress_test.js

```diff
@@ -23,6 +23,8 @@ exports.compress = {
       actual.push(entry.path);
     });
     parse.on('close', function() {
+      actual.sort();
+      expected.sort();
```

Comment:
correct but the order they are added isn't defined when doing `file` calls to archiver as it has to stat behind the scenes and only actually handles said file when that stat comes back successfully. ie this can make things get out of order.

now that stat data is passed in one of the latest commits though this is less of the case but again doesnt hurt much in the end.